Pipfile

```diff
@@ -0,0 +1,53 @@
+[[source]]
+
+name = "pypi"
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+
+
+[packages]
+
+amqp = ">=1.4.9"
+"aniso8601" = ">=1.2.1"
+anyjson = ">=0.3.3"
+arrow = ">=0.10.0"
+billiard = ">=3.3.0.23"
+celery = ">=3.1.25"
+celery-with-mongodb = ">=3.0"
+certifi = ">=2017.4.17"
+chardet = ">=3.0.4"
+click = ">=6.7"
+enum-compat = ">=0.0.2"
+eventlet = ">=0.21.0"
+"flake8" = ">=3.3.0"
+flask = ">=0.12.2"
+flask-cors = ">=3.0.2"
+flask-restful = ">=0.3.6"
+flask-socketio = ">=2.9.0"
+flask-sslify = ">=0.1.5"
+greenlet = ">=0.4.12"
+honcho = ">=1.0.1"
+icalendar = ">=3.11.4"
+idna = ">=2.5"
+isodate = ">=0.5.4"
+itsdangerous = ">=0.24"
+"jinja2" = ">=2.9.6"
+kombu = ">=3.0.37"
+markupsafe = ">=1.0"
+mccabe = ">=0.6.1"
```

Comment:
`flake8`, `mccabe`, `pycodestyle`, and `pyflakes` could go under `[dev-packages]`.

Reply:
Moved to #154